manifests/4.9/kubernetes-nmstate-operator.v4.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,219 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [{
+        "apiVersion": "nmstate.io/v1beta1",
+        "kind": "NMState",
+        "metadata": {
+          "name": "nmstate"
+        },
+        "spec": {
+          "nodeSelector": {
+            "beta.kubernetes.io/arch": "amd64"
+          }
+        }
+      }]
+    capabilities: Basic Install
+    certified: "false"
+    categories: OpenShift Optional
+    description: |
+      Kubernetes NMState is a declaritive means of configuring NetworkManager.
+    containerImage: quay.io/openshift/origin-kubernetes-nmstate-operator:4.9
+    createdAt: 2020-011-12 17:13:03
```

Comment:
Not sure if it's important, but the date seems off

Reply:
not important for ART builds to work, but it could certainly be changed if needed (i just copied 4.8 manifests dir + search&replace "4.8" by "4.9")

Reply:
Gotcha, not a must